{{cookiecutter.project_slug}}/.env.example

```diff
@@ -83,6 +83,11 @@ DB_USER='{{cookiecutter.project_slug}}'
 DB_PASS='!!!POSTGRES_PASSWORD!!!'
 DB_HOST='127.0.0.1'
 
+# TEST DB SETTINGS THESE MUST MATCH GH ACTIONS
```

Comment:
REALLY good code comments in this PR ✨ 
Not redundant or obvious and drawing the readers eyes to import nuances